src/l500/l500-private.h

```diff
@@ -377,19 +377,29 @@ namespace librealsense
             double APD_temperature;
             double HUM_temperature;
             double AlgoTermalLddAvg_temperature;
+
+            temperatures()
+                : LDD_temperature(.0)
```

Comment:
I'm used to seeing (and writing) `0.` but have never seen `.0`... not very important, up to you